apps/react/src/features/genres/components/GenresList/GenresList.tsx

```diff
@@ -0,0 +1,38 @@
+import { memo, FC } from 'react';
+import { Link } from 'react-router-dom';
+import { Genre } from '@js-camp/core/models/genre';
+import { List, ListItem, IconButton, ListItemText } from '@mui/material';
+import DeleteIcon from '@mui/icons-material/Delete';
+
+import { PATH_TO_GENRES } from '../../routes';
+
+import styles from './GenresList.module.css';
+
+type Props = {
+
+	/** Genres. */
+	readonly genres: Genre[];
+};
+
+/** Genres list.  */
+const GenresListComponent: FC<Props> = ({ genres }: Props) => (
```

Comment:
You can omit function argument type annotation when using `FC<Props>`
```suggestion
const GenresListComponent: FC<Props> = ({ genres }) => (
```

Reply:
Lint notifies of an error if we omit this type annotation